fineract-provider/src/main/java/org/apache/fineract/portfolio/paymenttype/data/PaymentTypeData.java

```diff
@@ -18,7 +18,12 @@
  */
 package org.apache.fineract.portfolio.paymenttype.data;
 
-public class PaymentTypeData {
+import org.apache.commons.lang.builder.EqualsBuilder;
```

Comment:
Again - consistency? Of course this does also work, so if you are short on time, we can leave all of these in, the other points raised are more important than these.